sphinx_wagtail_theme/layout.html

```diff
@@ -23,8 +23,8 @@
     {% block htmltitle %}
         <title>{{ title|striptags|e }}{{ titlesuffix }}</title>
     {% endblock %}
-    {%- if favicon %}
-        <link rel="shortcut icon" href="{{ pathto('_static/' + favicon, 1) }}" />
+    {%- if favicon_url %}
+        <link rel="shortcut icon" href="{{ pathto(favicon_url, 1) }}" />
```

Comment:
Looks like we might need to use `favicon_url` directly instead of using `pathto()`, as apparently this causes issues on pages other than the home page, see #281. Sorry, I should've tested a bit more!